Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+COPY requirements.txt /usr/src/app/
+WORKDIR /usr/src/app
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+COPY ampcrowd/docker-entrypoint.sh /usr/src/app/ampcrowd
+ENTRYPOINT ["bash", "ampcrowd/docker-entrypoint.sh"]
+CMD ["-d"]
```

Comment:
Ah, ok. I'll have to adjust to that philosophy.

 So the issue is that `debug` isn't precise enough, we really want three modes:
* running in the foreground, not debugging (for normal Docker operation)
* running as a daemon, not debugging (for normal non-Docker operation)
* running in the foreground, debugging (for debug mode, either Docker or non-docker).

So how about two flags, `docker` and `debug`, with the following behavior:
* If either flag is set, gunicorn should run in the foreground. Otherwise, it should run as a daemon.
* If `debug` is set, celeryd should not run as a daemon. Otherwise, it should.
* By default, we should run in `docker` but not `debug` mode (still not sure about `ssl`).